wavelength-plugins/tsumeta.js

```diff
@@ -115,6 +115,40 @@ exports.commands = {
 			write();
 		},
 
+		modify: "editproposal",
+		edit: "editproposal",
+		modifyproposal: "editproposal",
+		editproposal: function (target, room, user) {
+			if (!isTsuMetaCouncil(user.userid)) return this.errorReply('You are not in the TsuMeta council, or have been suspended.');
+			if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+			let parts = target.split(',');
+			for (let u in parts) parts[u] = parts[u].trim();
+			if (!parts[0] || !parts[1]) return this.parse('/tsumetahelp');
+			let proposal = parts[0];
+			let proposalid = toId(proposal);
+			if (!proposals[proposalid]) return this.errorReply(`This proposal doesn't exist!`);
+			if (proposal.length > 500) return this.errorReply("Please keep your changes to a maximum of 500 characters.");
+			let newDesc = parts[1];
+			if (user.userid === "xcmr" && Db.councilmember.has("xcmr")) {
+				proposals[toId(proposal)].desc = newDesc;
+				write();
+				this.sendReplyBox(`You have successfully modified proposal <strong>${proposals[proposalid].idea}'s</strong> description to:<br />${proposals[proposalid].desc}`);
+				if (Rooms('tsumetacommittee')) {
+					Rooms('tsumetacommittee').add(`|c|~TsuMeta Council|${user.name} has edited ${proposals[proposalid].idea}'s description.`).update();
+				}
+				return true;
+			}
+			if (proposals[proposalid].creator !== user.userid && user.userid !== "desokoro") return this.errorReply(`Only the creator "${proposals[proposalid].creator}" can modify this proposal.`);
```

Comment:
merge the xcmr check into this one.
`if (proposals[proposalid].creator !== user.userid && user.userid !== "desokoro" && (user.userid !== 'xcmr' || !Db.councilmember.has("xcmr")))`